docs/appendices/release-notes/unreleased.rst

```diff
@@ -69,6 +69,9 @@ None
 Changes
 =======
 
+- Added the scalar function ``CURRENT_TIME``, which returns the time portion
+  of ``CURRENT_TIMESTAMP`` in absolute milliseconds, see :ref:`current_time`.
```

Comment:
I think `CURRENT_TIME` could be explained without referencing to `CURRENT_TIMESTAMP` - would be easier to understand as you don't need to piggy-back onto existing knowledge of `CURRENT_TIMESTAMP`